src/kit.ts

```diff
@@ -386,10 +386,12 @@ const VsArchitectures: {[key: string]: string} = {
  * Preferred CMake VS generators by VS version
  */
 const VsGenerators: {[key: string]: string} = {
-  14: 'Visual Studio 14 2015',
-  15: 'Visual Studio 15 2017',
+  11: "Visual Studio 11 2012",
```

Comment:
Use `''` for strings (also applies to this one `"Visual Studio 12 2013",` below)

Reply:
`clang-format` has a fix for this one.